src/pages/assets/css/ImageButton.css

```diff
@@ -26,8 +26,8 @@
 }
 
 .img-component-container img {
-  max-width: 100%;
-  height: 100%;
+  width: auto;
+  height: auto;
```

Comment:
### Discussion
Are u sure about these changes? Keep in mind the breaking scenario at my laptop. 
If u have confusions about `auto` and `100%`: https://stackoverflow.com/questions/15943009/difference-between-css-height-100-vs-height-auto